codeql/windows-drivers/queries/suites/windows_driver_recommended.qls

```diff
@@ -29,4 +29,5 @@
       - Likely Bugs/Conversion/InfiniteLoop.ql
       - Likely Bugs/Memory Management/UseAfterFree/UseAfterFree.ql
       - Likely Bugs/UninitializedPtrField.ql
-      - Security/Crytpography/HardcodedIVCNG.ql
\ No newline at end of file
+      - Security/Crytpography/HardcodedIVCNG.ql
+      - Security/CWE/CWE-704/WcharCharConversionLimited.ql
```

Comment:
This file imports the Must-Fix file, so no need to duplicate including it here.  Did you mean to include the non-limited version as recommended?

Reply:
@raulgarciamsft Ping on this question.

Reply:
yes, that was the intent